Doc/library/asyncio-api-index.rst

```diff
@@ -19,10 +19,27 @@ await on multiple things with timeouts.
     :class: full-width-table
 
     * - :func:`run`
-      - Create event loop, run a coroutine, close the loop.
+      - Create an event loop, run one async function, and close the loop.
```

Comment:
It really runs a coroutine object not an async function. The previous wording was better.

Reply:
Agreed. The thing it runs is whatever the async function call returns, and such a thing is called a coroutine. We don't want people to think that they have to pass an async function name here.